features/claims/hardship_claims_banner.feature

```diff
@@ -9,7 +9,7 @@ Feature: A hardship claims banner will appear on the claims list, until the user
 
     And I click 'Start a claim'
     Then The hardship claims banner is visible
-    And the page should be accessible within "#content"
+    Then the page should be accessible within "#content"
```

Comment:
Very unimportant question. Is there a reason for changing this 'And' to a 'Then'?

Reply:
> Very unimportant question. Is there a reason for changing this 'And' to a 'Then'?

To keep all our accessibility assertions consistent with the step definition

Reply:
nit, not a blocker, the `Then` and `And` use don't need to match the step definition. Their use should be more inline with the "flow" of the language in the features scenario. but not important really